### PR TITLE
fix: fix separator when use window, it will make vite wrong

### PR DESCRIPTION
**Pull request by shuaiqidekaige**
platform： window

when use window：
image

and i find vite will resolve path:
image

we need resolve url when use window

### CODE REVIEW

**Comment by sanyuan0704** [comment]
You can run `pnpm lint:fix` to  resolve the lint error in Github CI.

File: src/node/utils.ts [hunk 1]
```diff
@@ -11,3 +11,7 @@ export const createHash = (info: string): string => {
   }
   return createHashFunc('sha256').update(info).digest('hex').slice(0, 8);
 };
+
+export const toSeq = (url: string, sep = '/') => {
```
Comment (sanyuan0704) [inline src/node/utils.ts:15]: How about rename it to `normalizePath`? It will be easier to understand.

Comment (shuaiqidekaige) [reply src/node/utils.ts:15]: Sounds good

Comment (shuaiqidekaige) [reply src/node/utils.ts:15]: > 怎么改名`normalizePath`？会更容易理解。

Sounds good

**Pull request merged**